assets/airports/eddm.json

```diff
@@ -1,883 +1,2695 @@
+            "name_offset": [[-1.5, -1.5], [1.5, -1.1]],
+            "end": [["N48.344783", "E11.804606", "1470ft"], ["N48.340656", "E11.750994", "1486ft"]],
+            "delay": [20, 30],
+            "ils": [true, true]
+        },
+        {
+            "name": ["26R", "08L"],
+            "name_offset": [[-1.5, 0.8], [1.5, 1.2]],
+            "end": [["N48.366886", "E11.821172", "1449ft"], ["N48.362753", "E11.767528", "1467ft"]],
+            "delay": [30, 20],
+            "ils": [true, true]
+        }
+    ],
+    "sids": {
+        "ALG": {
+            "icao": "ALG",
+            "name": "Allgäu",
+            "suffix": {"08L":"2Q", "08R":"2E", "26L":"2S", "26R":"2N"},
+            "rwy": {
+                "08L":  ["DM071", ["DM094", "S230"], "LAKOL", "OBAXA", ["MERSI", "S250"]],
+                "08R":  ["DM081", ["ALONU", "S230"], "LAKOL", "OBAXA", ["MERSI", "S250"]],
+                "26L":  [["MSE", "S220"], ["DM053", "A70|S250"], "AMEXO"],
+                "26R":  [["MNE", "S220"], ["DM061", "A70|S250"], "AMEXO"]
+            },
+            "exitPoints": {
+                "ALG": ["ALG"]
+            },
+            "draw": [
+                ["DM071", "DM094", "LAKOL"],
+                ["DM081", "ALONU", "LAKOL", "OBAXA", "MERSI", "ALG"],
+                ["MNE", "DM053", "AMEXO"],
+                ["MSE", "DM061", "AMEXO", "ALG"]
+            ]
+        },
+        "ANKER": {
+            "icao": "ANKER",
+            "name": "Anker",
+            "suffix": {"08L":"8Q", "08R":"9E", "26L":"7S", "26R":"9N"},
+            "rwy": {
+                "08L": ["DM070", "DM077", "DM079"],
+                "08R": [["DM083", "S230"]],
+                "26L": [["DM051", "S220"], ["DM056", "S250"], "DM057"],
+                "26R": [["DM066", "S220"], ["DM064", "S250"], "DM057"]
+            },
+            "exitPoints": {
+                "ANKER": [["ANKER", "S250"]]
+            },
+            "draw": [
+                ["DM070", "DM077", "DM079", "ANKER"],
+                ["DM083", "ANKER"],
+                ["DM051", "DM056", "DM057"],
+                ["DM066", "DM064", "DM057", "ANKER"]
+            ]
+        },
+        "BIBAG": {
+            "icao": "BIBAG",
+            "name": "Bibag",
+            "suffix": {"08L":"2Q", "08R":"2E", "26L":"2S", "26R":"2N"},
+            "rwy": {
+                "08L": [["DM071", "S230"], ["RATGI", "S250"]],
+                "08R": [["DM080", "S230"], ["RATGI", "S250"]],
+                "26L": [["DM049", "S210"], ["DM050", "S210"], "MUN", ["MEBEK", "S250"]],
+                "26R": [["DM067", "S210"], "DM050", ["MUN", "S250"], ["MEBEK", "S250"]]
+            },
+            "exitPoints": {
+                "BIBAG": ["BIBAG"]
+            },
+            "draw": [
+                ["DM071", "RATGI"],
+                ["DM080", "RATGI", "BIBAG"],
+                ["DM049", "DM050"],
+                ["DM067", "DM050", "MUN", "MEBEK", "BIBAG"]
+            ]
+        },
+        "BIBAG2W": {
```

Comment:
I see that you have `BIBAG2W` _and_ `BIBAG2S` from runway 26L. Are we able to remove one? Or do they actually use two different variants of the same arrival over the same fix going to the same runway?